src/components/CountryBoard.vue

```diff
@@ -1,7 +1,119 @@
-<script lang="ts" setup></script>
+<script lang="ts" setup>
+import { useCountriesStore } from "@/stores/countries.store";
+import { computed, onMounted } from "vue";
+import { ICountry } from "@/types/countries";
+
+const store = useCountriesStore();
+const isLoading = computed((): boolean => store.$state.isLoading);
```

Comment:
w pinii nie musisz się dobierać do danych poprzez state - możesz zrobić to przez store.isLoading, typowanie również jest zbędne, bo dobrze zadziała Ci tutaj inferencja typów

Reply:
no właśnie - jak myślisz, z czego może wynikać to, że dostajesz ComputedRef<never[]>?

Reply:
Wiesz co oznacza typ never?

Reply:
never oznacza, że to nic nie zwraca. Tak jakby nie ma wartości, które może zwrócić. Dlaczego tu nie potrafi zinferowac? nie wiem